packages/next-app/src/app/internalmanager/(components)/TokenSelect.tsx

```diff
@@ -156,15 +172,100 @@ function TokenModal({
     }
   }
 
+  const tokenExplorerUrl = buildExplorerTokenURL({
+    chainId: chain!.id,
+    tokenAddress: tokenSearch.toLowerCase() as `0x${string}`,
+  });
+
   useEffect(() => {
     if (!internalBalanceData?.user?.userInternalBalances) return;
     getWalletBalance(tokenAdresses as `0x${string}`[]);
   }, [internalBalanceData]);
 
+  useEffect(() => {
+    if (addressRegex.test(tokenSearch)) {
+      setIsTokenSearchDisabled(false);
+      if (
+        tokens.filter((token) => filterTokenInput({ tokenSearch, token }))
+          .length < 1
```

Comment:
```suggestion
        tokens.some((token) => filterTokenInput({ tokenSearch, token }))
```

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some